faq/index.md

```diff
@@ -2,153 +2,280 @@
 
-Rooftop installs are mostly to connect you to a supernode or a second hop from a supernode. Look at our [map](../map) to find line-of-sight to another node location less than two miles away. You can also talk to people in the neighborhood and convince them to also install a node. We also have a collection of rooftop panoramas where you can check for possible line-of-sight connections. 
+By joining the mesh you are obliged to share and extend the mesh in the same way that it was shared with you. At a minimum, this requires providing power to your rooftop router so that other members can connect to it. (It uses less power in a whole day than it takes to lightly toast a slice of bread). We may also request access to your rooftop after the initial installation in order to upgrade the rooftop router infrastructure. You are not obligated to pay a monthly fee although we request a recurring donation if you can afford it, to help keep the network running and growing. 
 
-The other type of install is where a directional Ubiquiti router is aimed at a nearby public space (park, cafe, bar etc.) as a public Wi-Fi access point. We currently have five public spaces with this type of install. 
+## <a name="volunteer"></a>How can I volunteer? What if I don’t know anything about networking?
 
-We support one outdoor router, the Ubiquiti Nanostation NSM5 (~$90). We have links for this on the [download page](../download)
+NYC Mesh welcomes people with diverse skill sets and from all backgrounds. Many of our volunteers started without any technical background. Join as at our monthly [meetup](https://www.meetup.com/nycmesh/) or reach out to us on [slack](https://slack.nycmesh.net/) to start a conversation about how you can contribute to our community. You can also read about some ways to help [here] (../help). 
 
-We can help you with the rooftop installation. Typically one or two routers are attached to a pole on the roof and an ethernet cable is run to the apartment, and connected to the TP-Link mesh router and an internet gateway. The install cost is usually $160 including labor and equipment. The ethernet cable also supplies the power to the router using "power over ethernet". If the roof (or balcony or window) is close to your apartment you might not need an additional mesh router.
+## <a name="neighbors"></a>I have neighbors who would also like to join NYC Mesh. Can we get connected together?
 
-## <a name="nsm5-indoor"></a>I want a NanoStation but I'm not ready for a rooftop install
+Yes! We prioritize installs for entire buildings and set up high-bandwidth point-to-point connections with dedicated hardware. If your building would like this dedicated service please email us, reach out to us on [slack](https://slack.nycmesh.net/), send us an email at install@nycmesh.net or call us directly at (646) 727-0866. Using this service can replace your current ISP for very large buildings.
```

Comment:
linkify email (mailto:) and phone number ([tel](https://stackoverflow.com/questions/1009181/url-scheme-for-phone-call))